src/LibVLCSharp.Forms/Platforms/Apple/OrientationHandler.cs

```diff
@@ -0,0 +1,42 @@
+﻿#if IOS
+using Foundation;
+using LibVLCSharp.Forms.Platforms.iOS;
+using LibVLCSharp.Forms.Shared;
+using UIKit;
+using Xamarin.Forms;
+
+[assembly: Dependency(typeof(OrientationHandler))]
+namespace LibVLCSharp.Forms.Platforms.iOS
+{
+    /// <summary>
+    /// Force orientation of iOS device.
+    /// In iOs client project, Developer should subscribe the AppDelegate.cs class to
+    /// all MessagingCenter below.
+    /// Refer to the sample LibVLCSharp.Forms.Sample.MediaElement for to see how to do it.
```

Comment:
>  Refer to the sample LibVLCSharp.Forms.Sample.MediaElement to see how to use it.

Reply:
I have to remove this comment beacause i moved all MessagingCenter to the library.